src/pybind/mgr/dashboard/module.py

```diff
@@ -103,6 +103,10 @@ def __init__(self):
     def shutdown(self):
         self._stopping.set()
 
+    @staticmethod
+    def str_to_bool(string):
```

Comment:
Use [distutils.util.strtobool](https://docs.python.org/2/distutils/apiref.html?highlight=distutils.util#distutils.util.strtobool) instead. The function will lower the string automatically.

Reply:
Thanks! Fixed that. See new commits